### PR TITLE
tested with and without credentials

### PR DESCRIPTION
**Pull request by pepgid**

### CODE REVIEW

File: wielder/util/terraform_credential_helper.py [hunk 1]
```diff
@@ -65,7 +65,18 @@ def get_aws_mfa_cred_command(role_name):
...
+    """
+    Looks for aws MFA session credentials file,
+    extracts env variables necessary for running terraform
+    :param role_name: role name to check against CLI MFA cache.
+    :return: returns env credential shell command or empty string.
+    """
+
+    env_cred = get_aws_mfa_cred(role_name)
 
     as_string_cmd = as_export_cmd(env_cred)
```
Comment (YaroslavPepticom) [inline wielder/util/terraform_credential_helper.py]: Rename something like:credential_to_cmd

File: wielder/util/terraform_credential_helper.py [hunk 2]
```diff
@@ -79,25 +90,25 @@ def get_aws_mfa_cred_command(role_name):
 if __name__ == "__main__":
```
Comment (YaroslavPepticom) [inline wielder/util/terraform_credential_helper.py]: move commented to test file

**Review by YaroslavPepticom — CHANGES_REQUESTED** [review]

File: wielder/util/terraform_credential_helper.py (hunk 2 above)
Comment (pepgid) [reply wielder/util/terraform_credential_helper.py]: fixed by moving to test. remember it is useful for cli

File: wielder/util/terraform_credential_helper.py (hunk 1 above)
Comment (pepgid) [reply wielder/util/terraform_credential_helper.py]: renamed and renamed module to better reflect the level of abstraction and reusability

**Pull request merged**